.rubocop.yml

```diff
@@ -1,3 +1,3 @@
 AllCops:
   DisplayCopNames: true
-  TargetRubyVersion: 2.5
+  TargetRubyVersion: 2.3
```

Comment:
Over at mutant I argued to instead drop mutant support for < 2.5. The extra work required and the tooling regressions (this change deactivates some tests) is not worth the extra effort. Given the fact that our commercial projects are all on 2.5.

If someone needs < 2.5 lets ask them to use the older releases of auom/mutant etc.